Libplanet.Explorer/Indexing/BlockChainIndexBase.cs

```diff
@@ -299,7 +299,7 @@ CancellationToken stoppingToken
 
             var blockDigest = store.GetBlockDigest(indexEnumerator.Current)!.Value;
             var txs = blockDigest.TxIds.Select(
-                txId => (ITransaction)store.GetTransaction(new TxId(txId))).ToArray();
+                txId => (ITransaction)store.GetTransaction(new TxId(txId))!).ToArray();
```

Comment:
From the looks of it, this should throw an `Exception` instead of trying to cast to make it clear that the query expects `store.GetTransaction()` to be non-`null`.